Sources/Mac/Extensions/Configuration+Imaginary.swift

```diff
@@ -7,7 +7,7 @@ public extension Configuration {
   }
 
   public static var transitionClosure: ((NSImageView, NSImage) -> Void) = { imageView, newImage in
-    guard let oldImage = imageView.image else {
+    guard let oldImage = imageView.image, imageView.window?.inLiveResize == false else {
```

Comment:
What is `inLiveResize` 🤔

Reply:
@onmyway133 if the window is being actively resized.

```
A Boolean value indicating whether the view is being rendered as part of a live resizing operation.
```

Reply:
👍